pandas/tests/frame/test_constructors.py

```diff
@@ -893,7 +893,7 @@ def create_data(constructor):
         [
             (lambda x: np.timedelta64(x, "D"), "m8[s]"),
             (lambda x: timedelta(days=x), "m8[us]"),
-            (lambda x: Timedelta(x, "D"), "m8[ns]"),
+            (lambda x: Timedelta(x, "D"), "m8[s]"),
             (lambda x: Timedelta(x, "D").as_unit("s"), "m8[s]"),
```

Comment:
```suggestion
            (lambda x: Timedelta(x, "D").as_unit("ms"), "m8[ms]"),
```

Otherwise it is the same as the case above now. Or just remove the case alltogether